internal/confmapprovider/discovery/bundle/bundle.d/receivers/oracledb.discovery.yaml.tmpl

```diff
@@ -0,0 +1,50 @@
+#####################################################################################
+#                               Do not edit manually!                               #
+# All changes must be made to associated .tmpl file before running 'make bundle.d'. #
+#####################################################################################
+oracledb:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)oracle"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)oracle" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)oracle"
+  config:
+    default:
+      endpoint: localhost:1521
+      username: otel
+      password: password
+      service: xe
+  status:
+    metrics:
+      successful:
+        - strict: oracledb.cpu_time
+          first_only: true
+          log_record:
+            severity_text: info
+            body: oracledb receiver is working!
+    statements:
+      failed:
+        - regexp: "connection refused"
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: The container is not serving http connections.
+        - regexp: "received goaway and there are no active streams"
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: Unable to connect and scrape metrics.
+      partial:
+        - regexp: 'invalid username/password'
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: >-
+              Make sure your user credentials are correctly specified using the
+              `--set splunk.discovery.receivers.oracledb.config.username="<username>"` and
```

Comment:
there are `{{ configProperty }}` and `{{ configPropertyEnvVar }}` template functions that will expand these for you.

Reply:
TY for these comments on my draft! Wasn't expecting that yet. Much appreciated! I'll be doing changes today but wanted to get this up as I'd gotten things "Passing" (although without all the proper conventions being followed yet).

Really appreciate these comments!!